x-pack/legacy/plugins/siem/public/pages/hosts/details/details_tabs.tsx

```diff
@@ -93,6 +95,10 @@ const HostDetailsTabs = React.memo<HostDetailsTabsProps>(
           path={`${hostDetailsPagePath}/:tabName(${HostsTableType.events})`}
           render={() => <EventsQueryTabBody {...tabProps} />}
         />
+        <Route
+          path={`${hostDetailsPagePath}/:tabName(${HostsTableType.alerts})`}
+          render={() => <HostAlertsQueryTabBody {...tabProps} defaultFilters={defaultFilters} />}
```

Comment:
I thought that we agree to use `pageFilters` instead of `defaultFilters` or do I miss something here?